src/components/SQForm/SQFormIconButton.tsx

```diff
@@ -1,17 +1,30 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-
 import {useFormButton} from './useFormButton';
 import {IconButton} from 'scplus-shared-components';
 
+interface SQFormIconButtonProps {
+  /** The Material UI Icon to render inside the button */
+  IconComponent: React.ElementType;
+  /** Custom disabled state */
+  isDisabled: boolean;
```

Comment:
should isDisabled, shouldRequireFieldUpdates, title, and type have the `?` optional flag since they weren't required in the PropTypes implementation?

Reply:
Lol i'm too slow. I'll remove my comment as it's a duplicate of yours

Reply:
![giphy (4)](https://user-images.githubusercontent.com/12816579/125635488-3826d8c8-dc78-434d-b6c5-3bcf57c2fbaa.gif)

Reply:
Correct @caleboleary @SeanGroff. Let me set these props optional. Thanks for pointing this out.